rbd/watchers_mimic_test.go

```diff
@@ -0,0 +1,160 @@
+// +build !luminous,!mimic
+//
+// Ceph Nautilus is the first version that supports watchers through librbd.
+
+package rbd
+
+import (
+	"testing"
+
+	"github.com/ceph/go-ceph/rados"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListWatchers(t *testing.T) {
+	conn, _ := rados.NewConn()
```

Comment:
I added a helper function to cut down on this test setup boilerplate (and it checks more error conditions). Your original patches probably predate that getting merged, but I'd prefer it get used to make the tests more uniform.

Reply:
I should be a tad bit more helpful and say the function name I'm referring to is `radosConnect`. :-)

Reply:
Ah, yes, I'll use that!